sys/cddl/contrib/opensolaris/uts/common/dtrace/dtrace.c

```diff
@@ -3270,7 +3278,7 @@ dtrace_dif_varstr(uintptr_t addr, dtrace_state_t *state,
  * dtrace_dif_variable() uses this routine as a helper for various
  * builtin values such as 'execargs'.
  */
-static uintptr_t
+static uintcap_t
```

Comment:
Note that the local variable `ret` here is still an `uintptr_t` so the return value will not be a valid capability.